src/spyglass/decoding/visualization.py

```diff
@@ -576,8 +577,9 @@ def create_interactive_2D_decoding_figurl(
     decode_view = create_2D_decode_view(
         position_time=position_info.index,
         position=position_info[position_name],
+        track_bin_centers=track_bin_centers,
+        track_bin_dimensions=track_bin_dimensions,
```

Comment:
Maybe also call this place_bin_size for consistency?

Reply:
Done, here and in the parameter name for the downstream function.